rotkehlchen/tests/api/test_calendar.py

```diff
@@ -14,6 +15,9 @@
 from rotkehlchen.types import ChecksumEvmAddress, SupportedBlockchain
 
 
+THREE_DAYS = DAY_IN_SECONDS * 3
+
+
```

Comment:
overkill. Used only in this test and only in 2 places. Just use the value directly

```suggestion
```